### ISSUE TITLE
No GitHub global release created for `@aws-crypto/*` packages

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-crypto-helpers#50]
I've been watching releases of `@aws-crypto/*` packages as seen in the screenshot below:
![Screen Shot 2020-02-07 at 12 20 07 PM](https://user-images.githubusercontent.com/16024985/74062977-5489c280-49a4-11ea-863c-fbb848f3d777.png)

However, I don't get notified of the new releases as releases happen per package.
Link: https://github.com/aws/aws-sdk-js-crypto-helpers/releases

Suggestion: Create a global GitHub release like one done by `@aws-sdk/*`
Example release v1.0.0-alpha.18 https://github.com/aws/aws-sdk-js-v3/releases/tag/v1.0.0-alpha.18

**Comment by seebees** [comment]
Yes, I want to get this done. Moving the CI/CD to GitHubActions and I want to work out all the steps that need to be added.
This will be added as part of the deployment process.
